modules/gcp-gke/inputs.tf

```diff
@@ -153,3 +154,13 @@ variable "skip_create_built_in_node_pool" {
    EOF
   type        = bool
 }
+
```

Comment:
Isn't this already added in line 144?

Reply:
I think this new variable 'zones' is already plugged through line 144 in this file. I think the one you wanted to add is in `modules/gcp-gke-node-pool/inputs.tf` or am I  wrong?